### ISSUE TITLE
bazel watcher does not work with `run -c`

### ISSUE THREAD

**Issue by Jdban** [bazelbuild/bazel-watcher#347]
iBazel - Version v0.11.1

`bazel` example
```
$ bazel run -c dbg SimpleExample
INFO: Invocation ID: 949321e5-6fe1-4c8a-bec4-0095c9e2b8da
INFO: Build option --compilation_mode has changed, discarding analysis cache.
INFO: Analyzed target //java/projects/Examples/SimpleExample:SimpleExample (0 packages loaded, 708 targets configured).
INFO: Found 1 target...
Target //java/projects/Examples/SimpleExample:SimpleExample up-to-date:
  bazel-bin/java/projects/Examples/SimpleExample/SimpleExample.jar
  bazel-bin/java/projects/Examples/SimpleExample/SimpleExample
INFO: Elapsed time: 0.638s, Critical Path: 0.37s
INFO: 1 process: 1 worker.
INFO: Build completed successfully, 5 total actions
INFO: Build completed successfully, 5 total actions
5 + 15 = 20
SimpleExample Successful
```

`ibazel` example: 
```bash
$ ibazel run -c dbg SimpleExample
iBazel [9:41AM]: Querying for files to watch...
Loading: 0 packages loaded
ERROR: Error while parsing 'buildfiles(deps(set(-c)))': syntax error at '- c )'
Loading: 0 packages loaded
Loading: 0 packages loaded
iBazel [9:41AM]: Bazel query failed: exit status 2
Loading: 0 packages loaded
ERROR: Error while parsing 'kind('source file', deps(set(-c)))': syntax error at '- c )'
Loading: 0 packages loaded
Loading: 0 packages loaded
iBazel [9:41AM]: Bazel query failed: exit status 2
iBazel [9:41AM]: Running -c
ERROR: Unrecognized option: -c
iBazel [9:41AM]: Error running Bazel exit status 2
```

**Comment by achew22** [comment]
@Jdban If you would like to add support for it (or any flag), please add it to the following list in a PR

https://github.com/bazelbuild/bazel-watcher/blob/master/ibazel/main.go#L42

That's the set of flags that are transparently passed through to Blaze.

**Comment by Jdban** [comment]
Oh awesome. I'll do that, thanks.

**Comment by achew22** [comment]
Thanks so much for sending in your first patch! I hope it wasn't too painful. Would you like a release made for this, or are you willing to wait for the normal one?

**Comment by Jdban** [comment]
I guess it would be nicer to have it sooner, but it's not an urgent need for us. Thanks for the offer though!

**Comment by achew22** [comment]
I just tagged and released `@bazel/ibazel@0.11.2` to npm.

Thanks for your contribution!
